firewall.go

```diff
@@ -195,8 +199,12 @@ func NewFirewall(l *logrus.Logger, tcpTimeout, UDPTimeout, defaultTimeout time.D
 		UDPTimeout:     UDPTimeout,
 		DefaultTimeout: defaultTimeout,
 		localIps:       localIps,
-		metricTCPRTT:   metrics.GetOrRegisterHistogram("network.tcp.rtt", nil, metrics.NewExpDecaySample(1028, 0.015)),
 		l:              l,
+
+		metricTCPRTT:          metrics.GetOrRegisterHistogram("network.tcp.rtt", nil, metrics.NewExpDecaySample(1028, 0.015)),
+		metricDroppedLocalIP:  metrics.GetOrRegisterCounter("firewall.dropped.local_ip", nil),
```

Comment:
I think it would make more sense to orient these on direction then dropped reason `firewall.{incoming, outgoing}.dropped.local_ip`, `firewall.{incoming, outgoing}.dropped.remote_ip`, `firewall.{incoming, outgoing}.dropped.no_rule`

Also seems like it opens us up to `firewall.{incoming, outgoing}.passed.conntrack` and `firewall.{incoming, outgoing}.passed.rule`